src/org/sosy_lab/java_smt/example/NQueens.java

```diff
@@ -0,0 +1,247 @@
+// This file is part of JavaSMT,
+// an API wrapper for a collection of SMT solvers:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Unlicense OR Apache-2.0 OR MIT
+
+package org.sosy_lab.java_smt.example;
+
+/**  This example program solves a NQueens problem of given size and prints a possible solution.
+ * <p>For example, the Queen can be placed in these ways for a field size of 4:
+ *  <pre>
+ *   .Q..
+ *   ...Q
+ *   Q...
+ *   ..Q.
+ * </pre>
+ */
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+import org.sosy_lab.common.ShutdownNotifier;
+import org.sosy_lab.common.configuration.Configuration;
+import org.sosy_lab.common.configuration.InvalidConfigurationException;
+import org.sosy_lab.common.log.BasicLogManager;
+import org.sosy_lab.common.log.LogManager;
+import org.sosy_lab.java_smt.SolverContextFactory;
+import org.sosy_lab.java_smt.SolverContextFactory.Solvers;
+import org.sosy_lab.java_smt.api.BooleanFormula;
+import org.sosy_lab.java_smt.api.BooleanFormulaManager;
+import org.sosy_lab.java_smt.api.Model;
+import org.sosy_lab.java_smt.api.ProverEnvironment;
+import org.sosy_lab.java_smt.api.SolverContext;
+import org.sosy_lab.java_smt.api.SolverContext.ProverOptions;
+import org.sosy_lab.java_smt.api.SolverException;
+public class NQueens {
```

Comment:
Before the class, an empty newline is missing. Please apply "ant format-source" for proper formatting.

Reply:
Maybe you have to reload the editor after applying the comand?